pysaliency/__init__.py

```diff
@@ -6,6 +6,7 @@
 from . import external_models
 from . import external_datasets
 from . import utils
+from . import http_models
```

Comment:
this still requires `orjson` to be installed. Let's simply not import http models by default. Or is there a different, more elegant pattern to avoid forcing the installation of `orjson`? Then I would be happy to go for it.